BootloaderCorePkg/Tools/GenCapsuleFirmware.py

```diff
@@ -427,7 +427,7 @@ def SignImage(RawData, OutFile, PrivKey):
     fwupdate_bin_file = 'fwupdate_unsigned.bin'
     open(fwupdate_bin_file, 'wb').write(unsigned_image + RawData)
 
-    rsa_sign_file(PrivKey, pubkey_file, fwupdate_bin_file, OutFile, True, True)
+    rsa_sign_file(PrivKey, pubkey_file, fwupdate_bin_file, OutFile, True, True, self.SIGN_HASH_TYPE)
```

Comment:
since we are  changing interface anyway, let us put important argument at the beginning.
In this case I think the type should be first parameter.

Reply:
I would add start params for all changed interfaces applicable.